README.md

```diff
@@ -1,17 +1,43 @@
-# Google CloudEvents – Python
+# Google CloudEvents - Python
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types used with Google services.
+At this moment the following types are available:
 
-## Prerequisites
+| Package | Python Class | Description |
+| ------------- | ------------- | ------------- |
+| google.events.cloud.pubsub.v1 | MessagePublishedEvent | This event is triggered when a Pub/Sub message is published. |
+| google.events.cloud.audit.v1 | AuditLogWrittenEvent | This event is triggered when a new audit log entry is written. |
 
-- Python 3.5 (or higher)
+## Installation and Usage
 
-## Generate
+**Note**: This library requires Python 3.7+.
 
-Generate libraries:
+To install this package, run
 
-```sh
-./gen.sh
+``` sh
+pip install --upgrade google-events
+```
+
+To use an event class, see the snippet below:
+
+``` python
+from google.events.cloud.pubsub.v1 import MessagePublishedEvent
+
+# Parses a Dict into an event
+# The Dict may be an argument in a background Cloud Function,
+# or the payload of an HTTP request received by a Cloud Run service with event
+# triggers.
+event = MessagePublishedEvent.from_dict(event_data)
+print(event.message)
 ```
 
-Observe the new files in `src/`.
\ No newline at end of file
+## Generation
```

Comment:
Let's put this part in `.github/CONTRIBUTING.md` so we hide it from the published package.

Reply:
Done.